lib/runctl.js

```diff
@@ -239,3 +239,12 @@ function requestOfTarget(req, rsp, callback) {
     callback(rsp);
   }
 }
+
+function clusterStatus() {
+  var mStatus = master.status();
+  // pst is used by PM, startTime is set by strong-cluster-control
+  mStatus.master.pst = mStatus.master.pst || mStatus.master.startTime;
+  mStatus.appName = agent().config.appName;
```

Comment:
Double checked. The config is loaded by `require()`ing the agent, even without starting profiling or enabling metrics, so this should be safe. Makes sense since it doesn't cause problems in the started message that was already using it.